ci/pipelines/pay-js.yml

```diff
@@ -0,0 +1,88 @@
+resources:
+  # GitHub Repos
+  - name: pay-js-pipeline-definition
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: master
+      paths:
+        - ci/pipelines/pay-js.yml
+#  - name: js-commons-git-release
+#    type: git
+#    icon: github
+#    source:
+#      uri: https://github.com/alphagov/pay-js-commons
+#      branch: master
+#      username: alphagov-pay-ci-concourse
+#      password: ((github-access-token))
+#      commit_filter:
+#        exclude: [ "\\[release\\]" ]
+  - name: js-metrics-git-release
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-js-metrics
+      branch: main
+      username: alphagov-pay-ci-concourse
+      password: ((github-access-token))
+      commit_filter:
+        exclude: [ "\\[release\\]" ]
+
+groups:
+#  - name: js-commons
+#    jobs:
+#      - version-and-push-commons
+  - name: js-metrics
+    jobs:
+      - version-and-push-metrics
+  - name: update-pay-js-pipeline
+    jobs:
+      - update-pay-js-pipeline
+
+
+jobs:
+  - name: version-and-push-metrics
+    plan:
+      - in_parallel:
+          - get: js-metrics-git-release
+            trigger: true
+      - task: build-and-version
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: node
+              tag: alpine3.16
+          params:
+            GITHUB_TOKEN: ((github-access-token))
+            REPO: alphagov/pay-js-metrics
+          inputs:
+            - name: js-metrics-git-release
+          run:
+            path: sh
+            dir: js-metrics-git-release
+            args:
+              - -c
+              - |
+                apk add --no-cache git github-cli
+                
+                git remote remove origin
+                git remote add origin https://$GITHUB_TOKEN@github.com/$REPO
```

Comment:
Can we make sure the `$GITHUB_TOKEN` isn't logged out anywhere visible

Reply:
It's not visible in the build output.